modules/images/src/lib/parse-image.js

```diff
@@ -0,0 +1,69 @@
+/* global Image, Blob, createImageBitmap, btoa, fetch */
+
+import {parseImageNode} from './node/parse-image-node';
+
+export const canParseImage = parseImageNode || typeof ImageBitmap !== 'undefined';
+
+// Parse to platform defined type (ndarray on node, ImageBitmap on browser)
+export function parseImage(arrayBuffer, options) {
+  if (parseImageNode(arrayBuffer, options)) {
+    return parseImageNode(arrayBuffer, options);
+  }
+
+  return parseToImageBitmap(arrayBuffer);
+}
+
+// Fallback for older browsers
+// TODO - investigate Image.decode()
+// https://medium.com/dailyjs/image-loading-with-image-decode-b03652e7d2d2
+export async function loadImage(url, options) {
+  if (parseImageNode) {
+    throw new Error('loadImage');
```

Comment:
Is the browser not supposed to use `loadImage`? What is the correct entry point?